use_cases/kg_chat/backend/src/main.py

```diff
@@ -38,28 +43,27 @@ class Payload(BaseModel):
 
 
 neo4j_connection = Neo4jDatabase(
-    host=os.environ.get("NEO4J_URL", "bolt://neo4j:7687"),
-    user=os.environ.get("NEO4J_USER", "neo4j"),
-    password=os.environ.get(
-        "NEO4J_PASS", "pleaseletmein")
+    host="bolt://neo4j:7687", user="neo4j", password="pleaseletmein"
 )
 
+
 openai_api_key = os.environ.get(
-    "OPENAI_API_KEY", "")
+    "OPENAI_API_KEY", "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"
```

Comment:
you're fired! :-D

Reply:
Revoked 🙃

Should never push something just before leaving work.